v2/processor.go

```diff
@@ -82,23 +83,55 @@ func NewProcessor(receiver Receiver, handler HandlerFunc, options *ProcessorOpti
 		}
 	}
 	return &Processor{
-		receiver:          receiver,
+		receivers:         []Receiver{receiver},
 		handle:            handler,
 		options:           opts,
 		concurrencyTokens: make(chan struct{}, opts.MaxConcurrency),
 	}
 }
 
-// Start starts the processor and blocks until an error occurs or the context is canceled.
+func (p *Processor) AddReceiver(receivers ...Receiver) {
```

Comment:
problem with this API is that you don't control the order in which it's called.
it it allowed to add a receiver after calling Start?

I would prefer making this part of the initialization of the Processor

Reply:
added `NewMultiProcessor()`